builtin/mainmenu/dlg_confirm_exit.lua

```diff
@@ -0,0 +1,53 @@
+-- Luanti
+-- Copyright (C) 2025 siliconsniffer
+-- SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+local function exit_dialog_formspec()
+	local show_dialog = core.settings:get_bool("enable_esc_dialog", true)
+	local formspec = {
+		"size[10,3,true]" ..
```

Comment:
Nitpick: Could use the newest formspec version here and drop the `,true` part (as real coordinates are then implied).

Doesn't really matter for this formspec, but might in the future, I guess?